style.css

```diff
@@ -98,4 +103,273 @@ li {
   cursor: pointer;
 }
 
+.vertical-space {
+  height: 50px;
+}
+
 /* hero section ends */
+
+/* works section starts */
+.works {
+  margin-top: 50px;
+  background-color: #f7f7f9;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: auto;
+  background-color: #fff;
+  height: 130vh;
+  border: 1px solid rgb(219, 216, 216);
+  border-radius: 16px;
+  padding: 16px;
+  margin: 25px;
+}
+
+.image-fluid {
+  width: 100%;
+  display: inline-block;
+  object-fit: contain;
+}
+
+.tonic {
+  color: var(--h3-color);
+  font-size: 26px;
+  font-family: var(--font-family);
+  font-weight: 700;
+  line-height: 24px;
+}
+
+.text {
+  margin-top: 10px;
+}
+
+.multi-post {
+  color: var(--h3-color);
+  font-size: 26px;
+  font-family: var(--font-family);
+  font-weight: 700;
+  line-height: 24px;
+}
+
+.flex-items {
+  display: flex;
+  margin-top: 10px;
+}
+
+.canopy {
+  list-style: none;
+  color: #344563 !important;
+}
+
+.works_paragraph {
+  color: var(--paragraph-color);
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.list-item {
+  margin-right: 25px;
+
+  /* background-color: var(--li-color); */
+  color: #7a869a;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+}
+
+.btn-container-flex {
+  margin-top: 5px;
+}
+
+.btn-sm {
+  background-color: #ebebff;
+  width: 50px;
+  text-align: center;
+  height: 25px;
+  border-radius: 5px;
+  border: none;
+  color: #b7bcff;
+}
+
+.btn-md {
+  background-color: #ebebff;
+  width: 60px;
+  text-align: center;
+  height: 25px;
+  border-radius: 5px;
+  border: none;
+  color: #b7bcff;
+}
+
+.btn-lg {
+  background-color: #ebebff;
+  width: 70px;
+  text-align: center;
+  height: 25px;
+  border-radius: 5px;
+  border: none;
+  color: #b7bcff;
+}
+
+.btn-container {
+  margin-top: 10px;
+}
+
+.btn-xxl {
+  width: 120px;
+  text-align: center;
+  height: 50px;
+  border: 1px solid #6070ff;
+  background-color: #fff;
+  color: #396df2;
+  border-radius: 10px;
+  font-weight: 500;
+  font-size: 17px;
+}
+
+.btn-sm:hover,
+.btn-md:hover,
+.btn-lg:hover,
+.btn-xxl {
+  cursor: pointer;
+}
+
+/* works section */
+.about {
+  height: 125vh;
```

Comment:
- [ ] I noticed an empty space at the end of this section also, just like I have said in one of the reviews above. For this case, I think it is because you have set a specific `height` on this section which makes it **rigid and fixed** that means even when the contents are few, it will still be of that height. This limits it from flowing with the content available. Please consider removing it hence it will take the height based on the content inside it. 😃 
![image](https://user-images.githubusercontent.com/90222110/177012295-bd5f1810-a699-4ec9-8475-6cc285369628.png)

Reply:
@BertrandConxy Thanks for the observation. I have corrected the extra spacing
![screenshot of project](https://user-images.githubusercontent.com/29442846/177139661-a1a9e7c8-271f-4ca8-a28f-9c405e60c8ef.png)
.